Clarify system database role in seeding (#1048)
neo4j/docs-operations: modules/ROOT/pages/clustering/seed.adoc

+
The cluster forms and the replicated Neo4j DBMS deployment comes online.

[NOTE]
====

> The system database defines the databases that should exist in your neo4j instance.

Reviewer: ```suggestion
The system database contains information about the databases that should exist in your Neo4j DBMS.
```

> If a database doesn't exist in your system database (because it hasn't been created previously) it will have to be created with `CREATE DATABASE <database-name>` even if it has been restored/loaded/imported.

Reviewer: ```suggestion
If a database does not exist in your system database (because it has not been created previously), it must be created with `CREATE DATABASE <database-name>` even if it has been restored/loaded/imported.
```
